tests/conftest.py

```diff
@@ -1879,6 +1885,9 @@ def core_dump_and_config_check(duthosts, request):
                     "inconsistent_config": inconsistent_config
                 }
             }
+            # Core dump and config check information
+            record_testsuite_property("dut_check_result", str(check_result))
```

Comment:
This probably will not work as expected. The idea we discussed is to add a capability to support record custom message for EACH test case in the test result file. By this way, it would be easier to figure out which tests contributed to inconsistent running config before&after test. The current implementation is to set some metadata for the current test suite. If we run multiple pytest scripts in a single pytest session, the recorded property would be overwritten by a later script that failed core dump or configdb check.

Another problem is that the full check_result would be too verbose to be recorded in test result as a piece of metadata.